azure-java-migration-copilot-client/src/main/java/com/azure/migration/java/copilot/command/ResourceCommand.java

```diff
@@ -26,6 +26,7 @@ public void execute(String commandText) {
         final String resources;
         try {
             resources = resourceFacade.listResource();
+            resourceFacade.initApplicationConfiguration();
```

Comment:
why initAppConfig here? what if user just give a folder only contains cf manifests file without source code? should we let user directly go the final step to generate bicep?

Reply:
What I am thinking is to prefullfill some field into the variables, so we can skip some set steps